src/test/java/in/projecteka/consentmanager/consent/service/ConsentManagerTest.java

```diff
@@ -60,15 +60,15 @@ public void askForConsent() {
         when (userClient.userOf(eq("chethan@ncg"))).thenReturn(Mono.just(new User()));
 
         String requestingHIUId = "hiu1";
-        StepVerifier.create(consentManager.askForConsent(requestingHIUId, consentDetail)).expectNextMatches(r -> r != null).verifyComplete();
+        StepVerifier.create(consentManager.askForConsent(requestingHIUId, requestedDetail)).expectNextMatches(r -> r != null).verifyComplete();
     }
 
     @Test
     public void askForConsentWithoutValidHIU() {
         HIPReference hip1 = HIPReference.builder().id("hip1").build();
         HIUReference hiu1 = HIUReference.builder().id("hiu1").build();
         PatientReference patient = PatientReference.builder().id("chethan@ncg").build();
-        ConsentDetail consentDetail = ConsentDetail.builder().hip(hip1).hiu(hiu1).patient(patient).build();
+        RequestedDetail requestedDetail = RequestedDetail.builder().hip(hip1).hiu(hiu1).patient(patient).build();
```

Comment:
I don't appreciate having Detail and other Suffix in the class name, but that's okay.

Reply:
we can call it RequestedInfo or so, but its about the same!